components/ImageInput/ImageInput.test.tsx

```diff
@@ -115,16 +116,21 @@ describe.each`
       filename: MOCK_FILE.name,
       key: MOCK_KEY,
       thumbnail_url: MOCK_THUMB,
-      full_url: "full.jpg",
+      full_url: MOCK_FULL_IMAGE,
     });
 
+    let expectedImage: string = MOCK_FULL_IMAGE;
+    if (type === "avatar") {
+      expectedImage = MOCK_THUMB;
+    }
```

Comment:
Nit: in this case, I think it'd be better to add the expected image URL as another property in the "table" under `describe.each` for both the `avatar` and `rect` variant for this input so that we can avoid writing the unnecessary logic in the tests and make it more linear/explicit to read